styles/dark-sans-serif.json

```diff
@@ -0,0 +1,42 @@
+{
+    "$schema": "https://schemas.wp.org/wp/6.2/theme.json",
+    "version": 2,
+    "title": "Dark Sans-serif",
+    "settings": {
+      "color": {
+        "palette": [
+            {
+                "color": "#222222",
+                "name": "Base",
+                "slug": "base"
+            },
+            {
+                "color": "#ffffff",
+                "name": "Contrast",
+                "slug": "contrast"
+            },
+            {
+                "color": "#222222",
+                "name": "Primary",
+                "slug": "primary"
+            },
+            {
+                "color": "#a2a2a2",
+                "name": "Secondary",
+                "slug": "secondary"
+            },
+            {
+                "color": "#303030",
+                "name": "Tertiary",
+                "slug": "tertiary"
+            }
+        ]
+      }
+    },
+    "styles": {
+        "typography": {
+          "fontFamily": "var:preset|font-family|inter"
```

Comment:
Should we expand on this to include the different font weights @matiasbenedetto @mikachan ?

Reply:
Inter has been removed from the theme; please see https://github.com/WordPress/twentytwentyfour/pull/72#issuecomment-1697734544

Reply:
ok. and also need to color palette update

Reply:
Thanks for suggestion @carolinan